config/initializers/paper_trail.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+PaperTrail.config.track_associations = true
+PaperTrail.config.has_paper_trail_defaults = {
+  on: %i(update destroy touch),
+  ignore: %i(created_at updated_at lock_version)
```

Comment:
still seeing the ignored columns in versions of  the models locally. I think this config needs to be moved inside the engine.rb file inside either the `config.before_initialize` block or add a `config.to_prepare do` block and add the config there.

Reply:
Yeah, on further review the `ignore` option is just setting a list of attributes whose changes should _not_ trigger creating a new version. See: https://github.com/paper-trail-gem/paper_trail#ignore.

I should probably just remove this line entirely, since it's unlikely to prevent the unwanted creation of superfluous new versions (which was my original intent):
* `created_at` typically only gets set once, shouldn't be getting changed
* `updated_at` would only be updated by an actual `update` or `touch` event, in which case we'd want to create a new version anyway
* `lock_version` should only be updated by an actual `update` event as well (I think?)

Reply:
I would still try adding the the lines to a config block inside engine.rb and if that works you can remove this file entirely.